pages/accused-witches-occupations.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div>
+    <div id="main-content" class="pl-5 pr-5 pt-3 pb-3">
+      <div class="flex content-start items-center">
+        <h1 class="text-sm md:text-xl lg:text-2xl">Accused Witches Occupations with Unknown Values</h1>
+        <span class="rounded-full border-r border-l border-gray-400 w-6 h-6 flex items-center justify-center ml-2">
+          <!-- icon by feathericons.com -->
+          <svg aria-hidden="true" class="" data-reactid="266" fill="none" height="24" stroke="#606F7B" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg" @click="showPageInfo()">
+            <line x1="12" y1="16" x2="12" y2="12"></line><line x1="12" y1="8" x2="12" y2="8"></line>
+          </svg>
+        </span>
+      </div>
+
+    </div>
+    <div>
+      <iframe style="width: 80vw; height: 70vh; border: none;" src="https://query.wikidata.org/embed.html#%23Social%20class%20of%20the%20accused%20witches%20in%20the%20Survey%20of%20Scottish%20witchcraft%0A%0A%23defaultView%3ABubbleChart%0ASELECT%20(IF(BOUND(%3Foccupation)%2C%20%3FoccupationLabel%2C%20%22unknown%22)%20as%20%3Flabel)%20%3Fcount%0A%3Foccupation%20%3FoccupationLabel%20%0A%0AWHERE%0A%7B%0A%20%20%7B%0A%20%20%20%20SELECT%20%3Foccupation%20(COUNT(%3Fperson)%20AS%20%3Fcount)%20WHERE%20%7B%0A%20%20%20%20%20%20%3Fperson%20wdt%3AP4478%20%3Faccused%20.%0A%20%20%20%20%20%20OPTIONAL%20%0A%20%20%20%20%20%20%7B%0A%20%20%20%20%20%20%3Fperson%20wdt%3AP106%20%3Foccupation.%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%20%20GROUP%20BY%20%3Foccupation%0A%20%20%7D%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%7D%0AORDER%20BY%20DESC(%3Fcount)%0ALIMIT%204000" referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups"></iframe>
+    </div>
+    <h1 class="text-sm md:text-xl lg:text-2xl">Occupations without Unknown Values</h1>
```

Comment:
I noticed that the header here isn't really positioned properly. We should only have one h1 header per page and this is the second. 

It might be nicer to restructure this page. Have a h1 saying "Accussed Witches Occupations". 

Underneath that we could have the info text as a paragraph or two. Then a h2 saying "Accused Witches Occupations with Unknown Values" and then this heading, also as a h2.

As this is a normal page, I don't think we need to have the info pop up. It was only on the main page because we had a lack of page estate because the map was full screen if that makes sense. Thanks Ruby

Reply:
have made these suggested changes, although still not sure on how it looks. Should h1 be made bigger and what font should h2 be?

Reply:
I was expecting us to have a proper heading scale in the CSS but it doesn't look like this is the case. h1 should be bigger and it would be good if the h2 used the same font as h1.

I think we need to set this up in global styles. We should use a heading scale to get the sizes right. [This site](https://baseline.is/tools/type-scale-generator/) should help you with setting one up. Thanks Ruby

Reply:
done

Reply:
i also made a change in the about page based on these heading changes

Reply:
Wow. What a difference this makes! This looks great thanks. The only issue I can see is that the grouping toggle on the map is a h3 so it is very big now. It shouldn't really be a h3 so would we be able to fix this? Think this will then be good to merge in. Thanks Ruby

Reply:
Thanks for fixing